<launch>

  <node name="chirp_controler" pkg="uav_commander" type="chirp_controler" ns="uav" output="screen">
    <rosparam command="load" file="$(find uav_commander)/config/params.yaml" />
  </node>



  <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure"/>
</launch>
